libs/elodin-editor/src/ui/command_palette/palette_items.rs

```diff
@@ -1513,6 +1583,64 @@ impl Default for PalettePage {
                 },
             ),
             toggle_body_axes(),
+            PaletteItem::new(
```

Comment:
Can we move this into a function (like`save_schematic`)? Especially since it's pretty long.

Reply:
I think Benoît asked me about lumping these together and I said, yes, so that's on me, but I agree in the future we ought to separate them.